src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -664,6 +666,561 @@ public static SyntaxToken XmlEntity(SyntaxTriviaList leading, string text, strin
+        /// <param name="content">A list of syntax nodes that represents the content of the xml multi line element.</param>
+        public static XmlElementSyntax XmlMultiLineElement(XmlNameSyntax name, SyntaxList<XmlNodeSyntax> content)
+        {
+            return XmlElement(
+                XmlElementStartTag(name),
+                content.Insert(0, XmlNewLine()).Add(XmlNewLine()),
+                XmlElementEndTag(name));
+        }
+
+        /// <summary>
+        /// Creates the syntax representation of an xml text that contains a newline token with a documentation comment 
+        /// exterior trivia at the end (continued documentation comment).
+        /// </summary>
+        public static XmlTextSyntax XmlNewLine()
+        {
+            return XmlText(XmlTextNewLine());
+        }
+
+        /// <summary>
+        /// Creates the syntax representation of an xml newline token with a documentation comment exterior trivia at 
+        /// the end (continued documentation comment).
+        /// </summary>
+        public static SyntaxToken XmlTextNewLine()
+        {
+            return XmlTextNewLine(true);
+        }
+
+        /// <summary>
+        /// Creates the syntax representation of an xml newline token for xml documentation comments.
+        /// </summary>
+        /// <param name="continueXmlDocumentationComment">
+        /// If set to true, a documentation comment exterior token will be added to the trailing trivia
+        /// of the new token.</param>
+        public static SyntaxToken XmlTextNewLine(bool continueXmlDocumentationComment)
+        {
+            SyntaxToken token = XmlTextNewLine(
+                TriviaList(),
+                Environment.NewLine,
+                Environment.NewLine,
+                TriviaList());
+
+            if (continueXmlDocumentationComment)
+                token = token.WithTrailingTrivia(DocumentationCommentExterior("/// "));
+
+            return token;
+        }
+
+        /// <summary>
+        /// Creates the syntax representation of an xml newline token for xml documentation comments.
+        /// </summary>
+        /// <param name="text">The raw text within the new line.</param>
+        /// <param name="continueXmlDocumentationComment">
+        /// If set to true, a documentation comment exterior token will be added to the trailing trivia
+        /// of the new token.</param>
+        public static SyntaxToken XmlTextNewLine(string text, bool continueXmlDocumentationComment)
+        {
+            var value = Environment.NewLine;
+            var token = new SyntaxToken(Syntax.InternalSyntax.SyntaxFactory.XmlTextNewLine((InternalSyntax.CSharpSyntaxNode)ElasticMarker.UnderlyingNode, text, value, (InternalSyntax.CSharpSyntaxNode)ElasticMarker.UnderlyingNode));
+
+            if (continueXmlDocumentationComment)
+                token = token.WithTrailingTrivia(DocumentationCommentExterior("/// "));
```

Comment:
I realize this is getting complicated, but the WithTrailingTrivia here replaces the trivia list completely, so the elastic trivia you added in the construction of the token is lost.  This will inhibit the formatter from adjusting the indentation of this line of the doc comment.  

What you want is more like this:

`
token = token.WithTrailingTrivia(token.TrailingTrivia + DocumentationCommentExterior("/// "));
`